microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode.md

```diff
@@ -45,33 +47,36 @@ Here are the main differences between the DevTools default user interface and Fo
 
 The **Activity Bar** is the most visible difference from the default user interface. Use it to open tools, help features, and settings.
 
-By default, the position of the **Activity Bar** adapts to the location of the DevTools window. If DevTools is undocked or docked to the bottom of the browser, the **Activity Bar** will be horizontal. If DevTools is docked to the side of the browser, the **Activity Bar** will be vertical, and tool names will not be visible. This enables quicker tool access when screen width is limited.
+By default, the position of the **Activity Bar** adapts to the location of the DevTools window:
+
+* When DevTools is undocked or docked to the bottom of the browser, the **Activity Bar** is horizontal.
+* When DevTools is docked to the side of the browser, the **Activity Bar** is vertical, and tool names aren't visible. This enables quicker tool access when screen width is limited.
 
-The **Activity Bar** top-aligned in Focus Mode, with DevTools docked:
+The **Activity Bar** left-aligned in Focus Mode, with DevTools docked to the side:
 
-![The Activity Bar top-aligned in Focus Mode, with DevTools docked](media/focus-mode-docked.png)
+![The vertical Activity Bar, with tool icons only](./focus-mode-images/docked.png)
 
 The **Activity Bar** top-aligned in Focus Mode, with DevTools popped out in a separate window:
 
-![The Activity Bar top-aligned in Focus Mode, with DevTools popped out in a separate window](media/focus-mode-undocked.png)
+![The horizontal Activity, with tool names and icons](./focus-mode-images/undocked.png)
```

Comment:
in capture, change from:
Removing tasks: jllhilxy2
to like:
Removing tasks: Test 2
Here is a clean model, it just names the task "task":
https://learn.microsoft.com/en-us/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension/get-started-images/devtools-extension-v211.png 

![image](https://user-images.githubusercontent.com/45407486/224152832-7be92b8e-2c18-4ba8-853a-ac5f869f6c93.png)

Reply:
Thanks for noticing this. I've actually removed those cryptic logs in the 3 images where they appeared.